Jenkinsfile

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env groovy
+
+REPOSITORY = 'signonotron2'
+DEFAULT_SCHEMA_BRANCH = 'deployed-to-production'
+DB_ADAPTERS = ['mysql', 'postgresql']
+
+node {
+  def govuk = load '/var/lib/jenkins/groovy_scripts/govuk_jenkinslib.groovy'
+
+  properties([
+    buildDiscarder(
+      logRotator(
+        numToKeepStr: '50')
+      ),
+    [$class: 'RebuildSettings', autoRebuild: false, rebuildDisabled: false],
+    [$class: 'ThrottleJobProperty',
+      categories: [],
+      limitOneJobWithMatchingParams: true,
+      maxConcurrentPerNode: 1,
+      maxConcurrentTotal: 0,
+      paramsToUseForLimit: 'Apps using "signonotron2_test" database',
+      throttleEnabled: true,
+      throttleOption: 'category'
+    ],
+  ])
+
+  try {
+    if (!govuk.isAllowedBranchBuild(env.BRANCH_NAME)) {
```

Comment:
I suspect this check is unnecessary because it's only required by applications that are used to test changes to the content schema, which I don't think is the case for Signonotron? Almost all projects are configured in Puppet so that they skip builds for release branches, which makes this check redundant.

The `DEFAULT_SCHEMA_BRANCH` variable above is unnecessary for the same reason.